Ray.jl/src/object_ref.jl

```diff
@@ -0,0 +1,14 @@
+struct ObjectRef
+    oid::ray_jll.ObjectIDAllocated
+end
+
+ObjectRef(hex_str::AbstractString) = ObjectRef(ray_jll.FromHex(ray_jll.ObjectID, hex_str))
+Base.string(obj_ref::ObjectRef) = String(ray_jll.Hex(obj_ref.oid))
```

Comment:
I think this should be something else; the julia docs say that `Base.string` and `Base.show` should be consistent, and they aren't in this case.

Reply:
ah actually I misread, its' _print_ that should be consistent with `string`.  I still dont' think we want to be `print`ing these as a string of hex digits without any additional disambiguation though

Reply:
I'll create an accessor called `hex_identifier` instead